packages/@vuepress/core/lib/node/App.js

```diff
@@ -348,7 +348,16 @@ module.exports = class App {
       computed: new this.ClientComputedMixinConstructor(),
       enhancers: this.pluginAPI.getOption('extendPageData').items
     })
-    this.pages.push(page)
+    const index = this.pages.findIndex(({ path }) => path === page.path)
+    if (index >= 0) {
+      // override a page if corresponding path already exists
+      if (!this.pages[index].overridable) {
+        logger.warn(`Override existing page ${chalk.yellow(page.path)}.`)
```

Comment:
If it's a flag to control the `overridable` of a page, should it wrap the `splice` operation?

Reply:
This is not a carefully considered API. So I decided to remove it at last.